app/Models/Facility.php

```diff
@@ -43,17 +44,14 @@ class Facility extends Model
     protected static function fieldValidator(string $field): string|array
     {
         return match ($field) {
-            'name' => 'required|string',
-            'url' => [
-                'bail',
-                'required',
-                'string',
-                'max:15',
+            'name' => Valid::trimmed(),
+            'url' => Valid::trimmed([
+                'max:30',
```

Comment:
To samo co w migracji, wydaje mi się, że 30 znaków to bardzo mało na URL. Dałbym ze 100.